pulp_rpm/tests/functional/api/test_sync.py

```diff
@@ -670,20 +669,24 @@ def test_sync_advisory_updated_update_date(init_and_sync):
 
 
 @pytest.mark.parallel
-def test_sync_advisory_older_update_date(init_and_sync):
+def test_sync_advisory_older_update_date(init_and_sync, rpm_advisory_api):
     """Test sync advisory with older update_date."""
     repository, _ = init_and_sync(url=RPM_UPDATED_UPDATEINFO_FIXTURE_URL)
     advisory_date = [
-        advisory["updated_date"]
-        for advisory in get_content(repository.to_dict())[PULP_TYPE_ADVISORY]
-        if advisory["id"] == RPM_ADVISORY_TEST_ID
+        advisory.updated_date
+        for advisory in rpm_advisory_api.list(
+            repository_version=repository.latest_version_href
+        ).results
+        if advisory.id == RPM_ADVISORY_TEST_ID
```

Comment:
I don't suppose we can use a query to grab this value?

Reply:
Yes, we can do direct filtering by ID. My bad.